playwright-e2e/tests/create-case.spec.ts

```diff
@@ -20,47 +22,14 @@ test.skip("Create a case", async ({ page }) => {
   );
   await signInPage.isSignedIn();
   // Add application details
-  // 2. Start new case, get case id and assert case id is created
+  // Start new case, get case id and assert case id is created
   await createCase.caseName();
   await createCase.createCase();
   await createCase.submitCase(createCase.generatedCaseName);
   await createCase.checkCaseIsCreated(createCase.generatedCaseName);
 
-  // 3. Orders and directions sought
+  // Orders and directions sought
   await startApplication.ordersAndDirectionsSoughtLink.isVisible();
   await startApplication.ordersAndDirectionsSoughtLink.click();
   await ordersAndDirectionSought.OrdersAndDirectionsHeading.isVisible();
-
```

Comment:
may be keep these comments for future implementations?

Reply:
@GauravChawlaMOJ this test isn't being used, and we could delete it all together, I removed the comments as it was messy and would likely change anyway.

Reply:
True